### DISCUSSION TITLE
控制中心提示「未找到页面」

### DISCUSSION THREAD

**Discussion by leirock** [ArtalkJS/Artalk#167] [Q&A] [answered]
前后端都是最新版，之前还是好的，想调试一下修改文章标题的部分，不明原因打开控制中心提示「未找到页面」。

然后直接清除所有数据库数据，新建一个数据库后依然这样，不知道是哪里出问题了。可以正常评论和加载评论，就是无法打开控制中心，但是可以正常用管理员密码登录。

<img width="416" alt="image" src="https://user-images.githubusercontent.com/19180725/169662548-3961b084-e556-4038-bd70-9aab7bd7530b.png">

**Comment by qwqcode** [comment, marked as answer]
ArtalkGo 内置缓存，数据库修改数据后，需要将缓存清空。可以侧边栏，点击页面列表上方的按钮清缓存。

你是反向代理了 ArtalkGo 后端程序吗？可以检查下 `后端地址\sidebar\index.html` 是否能访问，检查一下服务器反代配置，\sidebar 路径也需要反代，不仅仅是 \api 路径。

**Comment by qwqcode** [comment]
图中的界面不是 Artalk 的，可能是服务器面板的界面，可以检查下服务器面板配置

**Comment by leirock** [comment]
清除浏览器缓存和服务器的反代缓存以后正常了，谢谢啦～

**Reply by qwqcode** [reply]
😁😁
